server/api/users.js

```diff
@@ -523,9 +545,37 @@ router.post('/users/login', async (req, res, next) => {
   }
 });
 
-router.post('/users/logout', (req, res) => {
-  clearAuthCookies(req, res);
-  res.sendStatus(200);
+router.post('/users/logout', jwtAuth('read'), async (req, res, next) => {
+  try {
+    clearAuthCookies(req, res);
+    res.sendStatus(200);
+
+    if (req.user.user) {
```

Comment:
does `clearAuthCookies()` affect value of  `req.user.user` ?
If yes you might want to store it in a var first, if not just ignore me